src/app/authorization/login/login.component.ts

```diff
@@ -21,7 +21,7 @@ export class LoginComponent implements OnInit, OnDestroy {
   ) {}
 
   public formSubmitted(formValue:any) {
```

Comment:
why function is declared as public are we accessing in another class ? if not then please change it